roles/cis/tasks/3/3.5.1.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: "3.5.1 Ensure DCCP is disabled (Not Scored)"
+  vars:
+   modprobe_conf: /etc/modprobe.d/modprobe.conf
+  block:
+    - name: "verify if modprobe.conf is present"
+      shell: |
```

Comment:
should we use the ansible module copy with `force:no` instead of shell? you can set ownership and permission as well there

Reply:
same for directory with `file` module using `state:directory`

Reply:
Fixed